packages/experimental/Divider/src/Divider.styling.ts

```diff
@@ -0,0 +1,104 @@
+import { Theme } from '@fluentui-react-native/framework';
+import { layoutStyles } from '@fluentui-react-native/tokens';
+import { DividerTokens, DividerSlotProps, DividerAppearance } from './Divider.types';
+import { IconPropsV1 } from '@fluentui-react-native/icon';
+
+export const getDividerSlotProps = (tokens: DividerTokens, theme: Theme, icon?: IconPropsV1): DividerSlotProps => {
+  let iconProps: IconPropsV1 = {};
+  if (icon) {
+    if (icon.fontSource) {
+      iconProps = {
+        fontSource: {
+          ...icon.fontSource,
+          color: tokens.color || tokens.contentColor,
+        },
+      };
+    } else if (icon.svgSource) {
+      iconProps = {
+        svgSource: {
+          ...icon.svgSource,
+          color: tokens.color || tokens.contentColor,
+        },
+      };
+    }
+  }
+  return {
+    root: {
+      style: {
```

Comment:
You should use `buildProps` to construct these, so tokens can be cached. Look at other components' styling files for reference.